package.json

```diff
@@ -48,5 +48,8 @@
     "files": [
       "!**/fixtures/**"
     ]
+  },
+  "dependencies": {
+    "zod": "^3.21.4"
```

Comment:
This should be in the `package.json` files of both plugins. Currently, it would only be installed for people who clone this repo.